src/router/index.js

```diff
@@ -84,8 +84,13 @@ export default new Router({
 			}),
 		},
 		{
-			path: '/maps',
-			name: 'maps',
+			path: generateUrl('apps/maps', ''),
```

Comment:
you can use the generateurl directly in the Photos.vue file, there is no need to create new routes :)

Reply:
See other comments :)